src/loggers.jl

```diff
@@ -139,13 +140,24 @@ that prints to STDOUT.
 * `Logger`: the root logger.
 """
 function config(level::AbstractString; fmt::AbstractString=DEFAULT_FMT_STRING, levels=_log_levels, colorized=true)
-    _log_levels = levels
-    logger = Logger("root"; level=level, levels=levels)
-    handler = DefaultHandler(STDOUT, DefaultFormatter(fmt), Dict{Symbol, Any}(:is_colorized => colorized))
+    logger = Logger("root")
+    config(logger, level; fmt=fmt, levels=levels, colorized=colorized)
+end
```

Comment:
I would probably go with:
```julia
config(level::AbstractString; kwargs...) = config(Logger("root"), level; kwargs...)
config(logger::AbstractString, level::AbstractString; kwargs...) = config(Logger(logger), level; kwargs...)
```
So you don't have repeat the same kwarg defaults over and over